sdk/modelsrepository/Azure.Iot.ModelsRepository/samples/ModelsRepositoryClientSamples/ModelResolutionSamples.cs

```diff
@@ -50,10 +61,37 @@ public static async Task GetModelsFromGlobalRepoAsync()
             // In this case the above dtmi has 2 model dependencies.
             // dtmi:com:example:Thermostat;1 and dtmi:azure:DeviceManagement:DeviceInformation;1
             Console.WriteLine($"{dtmi} resolved in {models.Count} interfaces.");
+
+            #endregion Snippet:IoTModelsRepositorySamplesGetModelsFromGlobalRepoAsync
+        }
+
+        public static async Task GetMultipleModelsFromGlobalRepoAsync()
+        {
+            #region Snippet:IoTModelsRepositorySamplesGetMultipleModelsFromGlobalRepoAsync
+
+            var dtmis = new [] {"dtmi:com:example:TemperatureController;1", "dtmi:com:example:azuresphere:sampledevice;1"};
```

Comment:
move to just before it is used (line 80)

Reply:
Done.